lib/animas/animasLogic.js

```diff
@@ -15,9 +15,13 @@
  * == BSD2 LICENSE ==
  */
 
- Number.prototype.toFixedNumber = function(significant){
+ Number.prototype.toFixedNumber = function(significant, truncating){
```

Comment:
I guess there's no tests around the stuff in this file at present, so I don't think it's an absolute need. But I do think tests would help make the subtleties of what you're doing here easier to grok!

Related: maybe a different name for the `significant` parameter because unless I'm *not* understanding (possible!) it's not significant digits but rather number of decimal places, so maybe just `decimals` or similar? This is also not a distinction I would have remembered had I not looked it up recently (when the distinction between `toFixed` and `toPrecision` was relevant to me).